_sass/layout/_search.scss

```diff
@@ -22,10 +22,10 @@
   margin-top: 2 * $spacing-base;
 }
 
-.ais-Hits-item {
-  width: 100%;
-}
-
 .ais-RefinementList-labelText {
   margin-left: 0.5 * $spacing-base;
 }
+
+.hit-list {
+  font-size: $small-font-size;
```

Comment:
This makes the content in the hits look much better! Could we apply the same style to the Refinements list as well? It looks like those items still have the base Algolia tiny font size.